lib/Conversion/StandardToLLVM/ConvertStandardToLLVM.cpp

```diff
@@ -1054,14 +1054,15 @@ struct AllocOpLowering : public LLVMLegalizationPattern<AllocOp> {
     // Iterate strides in reverse order, compute runningStride and strideValues.
     auto nStrides = strides.size();
     SmallVector<Value, 4> strideValues(nStrides, nullptr);
-    for (auto indexedStride : llvm::enumerate(llvm::reverse(strides))) {
+    for (auto indexedStride : llvm::enumerate(strides)) {
```

Comment:
Actually, `enumerate` is useless here, the body of the loop never accesses `indexedStride.value()`. Could you just rewrite the loop to iterate on `index` to remove the confusion that led to the bug in the first place?